relay-server/src/services/processor/replay.rs

```diff
@@ -118,29 +109,32 @@ pub fn process(
     Ok(())
 }
 
+#[derive(Debug)]
+struct ReplayState<'a> {
```

Comment:
nit: We have similar structs for event and span normalization, where we call it `*Config`. Should we rename this to `ReplayProcessingConfig` or similar?